template/cmd_main.tmpl

```diff
@@ -1,29 +1,39 @@
 package main
 
 import (
-        _ "net/http/pprof"
+	"runtime"
+	_ "net/http/pprof"
 
-        "github.com/lileio/lile"
-        "github.com/lileio/logr"
+	"github.com/lileio/lile"
+	"github.com/lileio/logr"
 	"github.com/lileio/fromenv"
 	"github.com/lileio/pubsub"
 	"github.com/lileio/pubsub/middleware/defaults"
-        "google.golang.org/grpc"
-        "{{ .RelDir }}"
-        "{{ .RelDir }}/server"
+	"github.com/mattn/go-colorable"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"{{ .RelDir }}"
+	"{{ .RelDir }}/server"
 	"{{ .RelDir }}/{{ .Name }}/cmd"
 )
 
+func init() {
```

Comment:
Since logrus is global we'll only need to do this once, in the above commit is therefor fine!